src/main/java/ch/cern/eam/wshub/core/services/workorders/entities/WorkOrderActivityCheckListDefaultResult.java

```diff
@@ -0,0 +1,159 @@
+package ch.cern.eam.wshub.core.services.workorders.entities;
+
+import ch.cern.eam.wshub.core.annotations.InforField;
+import net.datastream.schemas.mp_fields.USERDEFINEDCODEID_Type;
+import org.openapplications.oagis_segments.DATETIME;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+public class WorkOrderActivityCheckListDefaultResult implements Serializable {
```

Comment:
Is this class necessary? Why not use the class from the proxy client/the Infor class?

Reply:
The class is definitely not necessary. It allows, however, the code in the filter method to be more concise.